### PR TITLE
Added new release docs and updated moving to Core for Java

### PR DESCRIPTION
**Pull request by jackdelahunt**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,18 +33,15 @@ node scripts/errors/generate_errors.js
...
-
-### GO
-- Go to the releases tab in Github and draft a new release
-- The tag should follow the format `app-services-sdk-go/v{Major-Minor-Path}`
-- The title of the release should follow the format `app-services-sdk-go app-services-sdk-go/v{Major-Minor-Path}`
-- Auto generate the patch notes making any changes you find sutible
-- Press `Publish release`
+Releasing a new version of the SDKs can be done by running a single workflow. 
+
+- Go to the Github Actions tab and select the `Java & Go SDK Release` workflow
```
Comment (andreaTP) [inline CONTRIBUTING.md:38]: `Java & Go SDK Release` rename to `Release`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -33,18 +33,15 @@ node scripts/errors/generate_errors.js
...
+
+- Select `Run workflow`, enter a version of the release in the format `X.Y.Z`
+
+- Select the branch the release will be created from and run the workflow
+
+- Once completed there will be a new release created with the name and tag `vX.Y,X` and a branch with the generated code needed named `release-X.Y.Z`
```
Comment (andreaTP) [inline CONTRIBUTING.md:44]:
```suggestion
- Once completed there will be a new release created with the name and tag `vX.Y.X` and a branch with the generated code needed named `release-X.Y.Z`
```
question, the tag is on the branch or on `main` ?

File: MOVING.md [hunk 3]
```diff
@@ -1,7 +1,12 @@
 # Moving to Core SDK
 
 ## What's changed
-While the code has moved the generation has stayed the same, which means updating to the Core SDK of the language you choose should be similar to updating to a new version in the old SDK.
+
+### Go
+While the code has moved the generation has stayed the same, which means updating to the Core SDK should be similar to updating to a new version in the old SDK
+
+### Java
+Java has been completly replaced with a new generator from Kiota you can learn more [here](https://microsoft.github.io/kiota/). This means any code in the previous Java SDK will no longer work and is no longer supported.
```
Comment (andreaTP) [inline MOVING.md:9]: this sounds a little harsh 😄 , the generated code looks pretty much similar and offers the functionality.
I would phrase it in a more "supportive" way, "try it out and ask questions and open issues if you find anything unexpected" ... something along those lines.

**Review by andreaTP — CHANGES_REQUESTED** [review]
a couple of comments, thanks for taking care!

File: CONTRIBUTING.md (hunk 1 above)
Comment (jackdelahunt) [reply CONTRIBUTING.md:38]: I wanted to be explicit about which SDKs we are supporting and releasing right now, then as we support them all just naming it `release` what do you think?

File: MOVING.md (hunk 3 above)
Comment (jackdelahunt) [reply MOVING.md:9]: Ah okay, again my knowledge on the new SDK is low so my thoughts on the Java SDK are not fully there until I get to try it with Rama. I will reword it as you know more about the transition

**Review by andreaTP — APPROVED** [review]
lgtm

**Pull request merged**